glossary.md

```diff
@@ -0,0 +1,161 @@
+# Adobe I/O Events Glossary
+
+[A](#a) &bull; [B](#b) &bull; [C](#c) &bull; [D](#d) &bull; [E](#e) &bull; [F](#f) &bull; [G](#g) &bull; [H](#h) &bull; [I](#i) &bull; [J](#j) &bull; [K](#k) &bull; [L](#l) &bull; [M](#m) &bull; [N](#n) &bull; [O](#o) &bull; [P](#p) &bull; [Q](#q) &bull; [R](#r) &bull; [S](#s) &bull; [T](#t) &bull; [U](#u) &bull; [V](#v) &bull; [W](#w) &bull; [X](#x) &bull; [Y](#y) &bull; [Z](#z)
+
+## A
+
+
+
+[Back to top](#adobe-io-events-glossary)
+
+## B
+
+
+
+[Back to top](#adobe-io-events-glossary)
+
+## C
+
+**Consumer:**
+
+**Consumer Org ID:**
+
+[Back to top](#adobe-io-events-glossary)
+
+## D
+
+
+
+[Back to top](#adobe-io-events-glossary)
+
+## E
+
+**Events:**
+
+**Event Code:**
+
+**Event Metadata:**
+
+**Event Provider:** Also known as "Provider"
+
+**Event Types:**
+
+[Back to top](#adobe-io-events-glossary)
+
+## F
+
+
+[Back to top](#adobe-io-events-glossary)
+
+## G
+
+
+
+[Back to top](#adobe-io-events-glossary)
+
+## H
+
+
+[Back to top](#adobe-io-events-glossary)
+
+## I
+
```

Comment:
I would favor documenting the integrationId/ApplicationId used in V1
and how they map with projectId and workspaceId

Reply:
IMS  and IMS OrgId would be a good addition too

Reply:
ingress

Reply:
IO Management API?